### DISCUSSION TITLE
编译h5后的结果和配置有出入

### DISCUSSION THREAD

**Discussion by bao2229402516** [NervJS/taro#12272] [Q & A] [answered]
`copy: {
    patterns: [
      { from: 'src/pages', to: 'dist/h5/pages' },
      { from: 'src/assets', to: 'dist/h5/assets' },
      { from: 'src/components', to: 'dist/h5/components' },
    ],
    options: {
      ignore: ['*.js', '*.css', '*.scss', '*.tsx', '*.ts', '*.json']
    }
  },`
编译成功后并没有排除掉对应的文件。
![image](https://user-images.githubusercontent.com/58587851/184049395-ab59d32e-5a59-4f1d-9ee5-0fa0e3ada477.png)

**Comment by ZakaryCode** [comment, marked as answer]
请参考官方[文档](https://webpack.js.org/plugins/copy-webpack-plugin/#globoptions)
